www/source/blog/2018-03-21-0550-Release.html.md

```diff
@@ -0,0 +1,161 @@
+- Document how to evaluate a release candidate [\#4768](https://github.com/habitat-sh/habitat/pull/4768) ([christophermaier](https://github.com/christophermaier))
+- add note to release doc about killing non release builds after tag push [\#4767](https://github.com/habitat-sh/habitat/pull/4767) ([mwrock](https://github.com/mwrock))
+- add note to release doc about killing non release builds after tag push [\#4767](https://github.com/habitat-sh/habitat/pull/4767) ([mwrock](https://github.com/mwrock))
+- \[studio\] Update stage1 tarball. [\#4766](https://github.com/habitat-sh/habitat/pull/4766) ([fnichol](https://github.com/fnichol))
+- Complete documentation coverage of all templating variables [\#4760](https://github.com/habitat-sh/habitat/pull/4760) ([christophermaier](https://github.com/christophermaier))
+- Encrypt and upload secrets. [\#4706](https://github.com/habitat-sh/habitat/pull/4706) ([elliott-davis](https://github.com/elliott-davis))
+- do not check mounts entering windows container studio [\#4668](https://github.com/habitat-sh/habitat/pull/4668) ([mwrock](https://github.com/mwrock))
+- pkg-export-k8s: Remove redundant newline filtering code [\#4648](https://github.com/habitat-sh/habitat/pull/4648) ([zeenix](https://github.com/zeenix))
+- Add release notes 0.54.0 [\#4644](https://github.com/habitat-sh/habitat/pull/4644) ([tashimi](https://github.com/tashimi))
+- Add asp.net core tutorial [\#4624](https://github.com/habitat-sh/habitat/pull/4624) ([mwrock](https://github.com/mwrock))
+
+## Bug fixes
+
+- Document installing and running private packages from builder [\#4733](https://github.com/habitat-sh/habitat/issues/4733)
+- hab-studio on linux should not require root/sudo when using docker [\#4716](https://github.com/habitat-sh/habitat/issues/4716)
+- windows packages are not honoring `pkg\_svc\_user` [\#4685](https://github.com/habitat-sh/habitat/issues/4685)
+- \\[www\\] Fix no-results message view [\#4591](https://github.com/habitat-sh/habitat/issues/4591)
+- Adding new values to default.toml don't get rendered in package updates [\#4576](https://github.com/habitat-sh/habitat/issues/4576)
+- builds fail when BusyBox `mount` exits non-zero [\#4391](https://github.com/habitat-sh/habitat/issues/4391)
+- Builds from web UI and CLI behave differently [\#4317](https://github.com/habitat-sh/habitat/issues/4317)
+- Updated default.toml files are seen on package update [\#4777](https://github.com/habitat-sh/habitat/pull/4777) ([christophermaier](https://github.com/christophermaier))
+- Set PATH metadata file to only contain my bin dirs [\#4722](https://github.com/habitat-sh/habitat/pull/4722) ([elliott-davis](https://github.com/elliott-davis))
+- use join for x-plat friendly root paths [\#4713](https://github.com/habitat-sh/habitat/pull/4713) ([mwrock](https://github.com/mwrock))
+
+**Closed issues:**
```

Comment:
Also minor, but I'd just make these two headings (sans colons)

```
## Closed Issues
## Merged Pull Requests
```

like the others